util/stringUtils.ts

```diff
@@ -18,4 +18,6 @@ export const parseRichTextIntoPlainText = (richText: string) =>
     // replaces emppty lines or lines just with spaces with an empty string
     .replace(/^\s*\n/gm, '')
     // replaces leading and trailing spaces from each line with an empty string
-    .replace(/^[ ]+|[ ]+$/gm, '');
+    .replace(/^[ ]+|[ ]+$/gm, '')
```

Comment:
calling attention to the fact this function as no tests, yet https://github.com/nodejs/nodejs.org/blob/main/util/__tests__/stringUtils.test.mjs exists

Reply:
Agreed. Would love to see more tests being added!